site/content/docs/getting-started/pre-requisites.md

```diff
@@ -1,41 +1,48 @@
 
-The following sections show you how to set up your environment to run the deployment tool.
+* The local credential profile can be configured by a variety of tools. For example, the credential profile can be configured with the [AWS Toolkit for Visual Studio](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/credentials.html) or the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), among others.
 
-## Node.js<a name="deployment-tool-setup-node"></a>
+  > Note: You need to make sure to add the appropriate AWS permissions to your credentials’ profile / assumed role. See [Setting up Credentials](setup-creds.md)
 
-The deployment tool requires the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/), and the AWS CDK requires [Node.js](https://nodejs.org/en/download/) version 10.13.0 or later (excluding versions 13.0.0 through 13.6.0). To see which version of Node.js you have installed, run the following command at the command prompt or in a terminal:
+#### .NET Core 3.1 or later
+* .NET CLI - the deployment tool can be  used from the .NET command-line interface (CLI) - a cross-platform toolchain for developing, building, running, and publishing .NET applications.
+
+* The .NET CLI is included with the [.NET SDK](https://docs.microsoft.com/en-us/dotnet/core/sdk). For information about how to install or update .NET, see [https://dotnet.microsoft.com/](https://dotnet.microsoft.com/).
+
+* The application you are deploying  must be built from .NET Core 3.1 or later (for example, .NET Core 3.1, .NET 5.0, .NET 6.0). To see what version you have, run the following on the command prompt or in a terminal:
 
 ```
-node --version
+dotnet --version
 ```
 
-**Note**  
-If the AWS CDK isn't installed on your machine or if the AWS CDK that's installed is earlier than the required minimum version (2.13.0), the deployment tool will install a temporary and "private" copy of the CDK that will be used only by the tool, leaving the global configuration of your machine untouched.
-If instead you want to install the AWS CDK, see [Install the AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_install) in the [AWS Cloud Development Kit (CDK) Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/)
+#### Node.js
 
-## .NET Core and .NET<a name="deployment-tool-setup-env-dotnet"></a>
+* The deployment tool requires the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/latest/guide/), and the AWS CDK requires [Node.js](https://nodejs.org/en/download/). AWS CDK requires Node.js version 10.13.0 or later (excluding versions 13.0.0 through 13.6.0), however we recommend installing the latest LTS version.

```

Comment:
CDK recently dropped support for Node 12 https://github.com/aws/aws-cdk/releases/tag/v1.160.0 

We should update this to say Node.js version 14 or later.

Reply:
done